SQL/0000-00-01-Permission.sql

```diff
@@ -50,69 +50,66 @@ CREATE TABLE `user_perm_rel` (
 
 INSERT INTO `permissions` VALUES
     (1,'superuser','Superuser - supersedes all permissions','1'),
-    (2,'user_accounts','User management','2'),
-    (3,'user_accounts_multisite','Across all sites create and edit users','2'),
-    (4,'context_help','Edit help documentation','2'),
+    (2,'user_accounts','User Accounts Module View Own Site','2'),
+    (3,'user_accounts_multisite','User Accounts Module create and edit users All Sites','2'),
+    (4,'context_help','Help Module Edit Documentation','2'),
     (5,'bvl_feedback','Behavioural QC','1'),
-    (6,'imaging_browser_qc','Edit imaging browser QC status','2'),
-    (7,'mri_efax','Edit MRI Efax files','2'),
-    (8,'send_to_dcc','Send to DCC','2'),
-    (9,'unsend_to_dcc','Reverse Send from DCC','2'),
-    (10,'access_all_profiles','Across all sites access candidate profiles','2'),
+    (6,'imaging_browser_qc','Imaging Browser Module Edit QC Status','2'),
+    (8,'send_to_dcc','Candidate Profile Module Send to DCC All Sites','2'),
+    (9,'unsend_to_dcc','Candidate Profile Module Reverse Send from DCC own Site','2'),
```

Comment:
why is one `All sites` and the other `Own site` is that actually the case ?

Reply:
Also the module is technically instrument_list not candidate_profile

Reply:
I don't know